src/main/scala/microsites/MicrositeKeys.scala

```diff
@@ -167,14 +201,64 @@ trait MicrositeAutoImportSettings extends MicrositeKeys {
 
   lazy val fr = new FileReader
 
+  def createVersionsJson(targetDir: String, content: List[Version]): File = {
+    val jekyllDir  = "jekyll"
+    val targetPath = s"$targetDir$jekyllDir/_data/versions.json"
+    createFile(targetPath)
+    writeContentToFile(content.asJson.toString, targetPath)
+    targetPath.toFile
+  }
+
+  def generateVersionList(versionStringList: List[String], ownVersion: String) = {
+    versionStringList
+      .map(version => {
```

Comment:
The braces wouldn't be needed

Reply:
Addressed in https://github.com/47deg/sbt-microsites/pull/379/commits/22cdc14758ba83507df43b91a8b6189d20f155a1